hebiros_gazebo_plugin/plugin_hebiros/hebiros_gazebo_group.h

```diff
@@ -7,41 +7,50 @@
 #include "hebiros/SettingsMsg.h"
 #include "hebiros/SetCommandLifetimeSrv.h"
 #include "hebiros/SetFeedbackFrequencySrv.h"
-#include "hebiros_gazebo_joint.h"
+#include "joint.h"
 
-using namespace hebiros;
+namespace hebi {
+namespace sim {
+namespace plugin {
 
 class HebirosGazeboGroup : public std::enable_shared_from_this<HebirosGazeboGroup> {
 
 public:
+  HebirosGazeboGroup(std::string name,
+    const std::vector<hebi::sim::Joint*>& joints_,
+    std::shared_ptr<ros::NodeHandle> n);
 
-  // TODO: Make these private.
+  void UpdateFeedback(const ros::Duration& iteration_time);
+
+  ros::Time GetPrevTime() { return prev_time; }
+  void SetPrevTime(ros::Time t) { prev_time = t; }
+
+private:
```

Comment:
Feel free to do this in a different PR, but we need to underscore these names (e.g., `name_`). It confused me above seeing `joints` before realizing they're defined here...

Reply:
Ack!  Good catch.  I thought I got everything.  I'll do that now.